src/pages/home/report/ReportActionItem.js

```diff
@@ -22,4 +23,6 @@ const ReportActionItem = props => (
 
 ReportActionItem.propTypes = propTypes;
 
-export default ReportActionItem;
+export default memo(ReportActionItem, (prevProps, nextProps) => (
+    prevProps.displayAsGroup === nextProps.displayAsGroup && _.isEqual(prevProps.action, nextProps.action)
+));
```

Comment:
Should we be worried about memory storage? Is it worth setting a limit?

Reply:
Hmm, not too sure what you mean by `maxSize`. `React.memo()` afaik does not store anything.

Reply:
Nevermind, I confused React.memo with a memoize library when researching the topic